design/one-pager-resource-status-improvements.md

```diff
@@ -0,0 +1,91 @@
+
+## Proposals
+Quote from `api-conventions` document in `sig-architecture` [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) about conditions:
+>Conditions should be added to explicitly convey properties that users and components care about rather than requiring those properties to be inferred from other observations. Once defined, the meaning of a Condition can not be changed arbitrarily - it becomes part of the API, and has the same backwards- and forwards-compatibility concerns of any other part of the API.
+
+To paraphrase, the conditions should, to the best of their ability, give the user and other components a precise status of the resource. A resource's conditions are also part of it's API, so changes should be well thought through.
+
+These proposals try to make the API for resources managed by crossplane providers to give users the best possible status about the health of their resources. 
+
+### 1: Improve Clarity of `Ready` Condition
+#### Problem
+If a user updates a managed resource in Crossplane today, there is no condition indicating this. The resource remains `Available` and no attempt is made at immediately observing the resource to check if it matches the desired state. Even if the update fails, the resource will be displayed as `Available`. 
+This means that a user cannot trust that an `Available` resource is a compliant resource, which can cause misinterpretation of the resource's state.
+
+Here is a diagram showing how the `Ready` condition is updated throughout the reconciler:
+##### Current `Ready` Condition Update Behavior
+```mermaid
+graph LR
+    Start(Reconcile Start)
+
+    NeedsDelete{Needs deleting?}
+    NeedsCreate{Needs creation?}
+    UpToDate{Up to date?}
+
+    Delete(Deletes the resource. \nReconciles immediately to make sure it is deleted. \nSets Ready: \nStatus = False \nReason = Deleting)
+    Create(Creates the resource. \nReconciled immediately to make sure it is created. \nSets Ready: \nStatus = False \nReason = Creating)
+    Observe(Doesn't need to do anything and doesn't change Ready. \nProvider author has to manually set Ready)
+    Update(Updates the resource. \nDoesn't reconcile immediately. \nDoesn't change ready status, so it remains `Ready = True`)
+
+    Start --> NeedsDelete
+    NeedsDelete -->|Yes| Delete
+    NeedsDelete -->|No| NeedsCreate
+    NeedsCreate -->|Yes| Create
+    NeedsCreate -->|No| UpToDate
+    UpToDate -->|Yes| Observe
+    UpToDate -->|No| Update
+```
+
+#### Solution
+Implement an `Updating` status, mimicking the `Creating` status. Setting this status every time an update request either succeeds or fails, informing a user that an update request has been requested, but we have not verified if the resource is compliant with the updated desired state.
```

Comment:
I read this part too, and I agree. 
As I was basing my proposal on updating the existing `Ready` condition, and I wanted it to be more about showing that it wasn't ready yet, I went with `Updating`, so it would match the `Creating` reason.

If the solution ends up being creating another condition, then I don`t mind following the API conventions linked.